qlty-plugins/plugins/linters/gitleaks/plugin.toml

```diff
@@ -1,7 +1,7 @@
 config_version = "0"
 
 [plugins.releases.gitleaks]
-github = "zricethezav/gitleaks"
```

Comment:
Is this change related?

Reply:
Not related, but it was a one line fix for Gitleaks to get it to stop crashing tests while I was testing.
 I can move this into a separate PR.